src/gt4sd/algorithms/generation/pgt/core.py

```diff
@@ -168,6 +171,38 @@ def get_generator(self, resources_path: str, **kwargs) -> Generator:
             num_return_sequences=self.num_return_sequences,
         )
 
+    @classmethod
+    def get_filepath_mappings_for_training_pipeline_arguments(
+        cls, training_pipeline_arguments: TrainingPipelineArguments
+    ) -> Dict[str, str]:
+        """Ger filepath mappings for the given training pipeline arguments.
+
+        Args:
+            training_pipeline_arguments: training pipeline arguments.
+
+        Returns:
+            a mapping between artifacts' files and training pipeline's output files.
+        """
+
+        if isinstance(training_pipeline_arguments, LanguageModelingSavingArguments):
+
+            if training_pipeline_arguments.ckpt is not None:
+
+                convert_pl_to_hf(training_pipeline_arguments)
```

Comment:
I like it a lot but I do not like the fact we produce files that are then simply moved, but for the time being is ok